### PR TITLE
SAMZA-1905: Added case study for eBay, Optimizely, Redfin and TripAdvisor

### PR DESCRIPTION
**Pull request by weisong44**
As per subject

### CODE REVIEW

File: docs/_case-studies/redfin.md [hunk 1]
```diff
@@ -23,8 +23,40 @@ excerpt_separator: <!--more-->
...
+communicate with Redfin’s customers, notification includes recommendations, instant 
+emails, scheduled digests and push notifications. Thousands of emails are delivered 
+to customers every minute at peak. 
+
+The notification system used to be a monolithic system, which served the company 
+well. However, as business grew and requirements evolved, it became harder and 
+harder to maintain and scale. The engineering team at Redfin decided to replace 
+the existing system with Samza primarily for Samza’s performance, scalability, 
+support for stateful processing and Kafka-integration. A multi-stage stream 
+processing pipeline was developed. Listings are first matched with customers and
```
Comment (vjagadish1989) [inline docs/_case-studies/redfin.md]: Any chance we can provide a diagram from their slides and add more technical details?

File: docs/_case-studies/tripadvisor.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+engineering team were faced with a few challenges
+
+-   Long lag time to downstream that is business critical
+-   Difficult to debug and troubleshoot due to scripts, environments, etc.
+-   Adding more nodes doesn’t help to scale
+ 
+The engineering team at TripAdvisor decided to replace the Hadoop solution 
+with a multi-stage Samza pipeline. In the new solution, after raw data is 
+first collected by Flume and ingested through a Kafka cluster, it is first 
+parsed, cleansed and partitioned; then processing logic such as windowing,
```
Comment (vjagadish1989) [inline docs/_case-studies/tripadvisor.md]: any chance we can expand on the solution a bit more? ie, an arch. diagram that also provides how a user in the community could build something similar?

File: docs/_case-studies/tripadvisor.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+-   Processing time is reduced from 3 hours to 1 hour
+-   Hardware requirement is reduced to ⅓ thanks to optimized usage
+-   Much simpler to debug and test the solution
+ 
+Key Samza features: *Stateful processing*, *Windowing*, *Kafka-integration*
+
+More information
+
+-   [https://www.youtube.com/watch?v=KQ5OnL2hMBY](https://www.youtube.com/watch?v=KQ5OnL2hMBY)
+
```
Comment (bharathkk) [inline docs/_case-studies/tripadvisor.md]: How about removing the additional line or maybe reduce the font size?
More information seems pronounced on the website compared to actual contents.

Same comment for all the case studies.

**Review by vjagadish1989 — APPROVED** [review]
approved.

**Pull request closed without merging**